docs/faq.md

```diff
@@ -0,0 +1,18 @@
+# Frequently Asked Questions 
+
+#### Why did we use Alevin-fry for processing? 
```

Comment:
General comment: I'd pick either Cell Ranger `count` or `cellranger count` but don't mix and also consider how you're talking about Alevin-fry in comparison (no code blocks).